frontend/src/routes/search/+page.svelte

```diff
@@ -24,6 +25,30 @@
 
   let modal: { top: number; left: number; site: DisplayedWebpage } | undefined;
 
+  const keybind = new Keybind([
+    { key: Keys.J, callback: searchCb.focusNextResult },
+    { key: Keys.ARROW_DOWN, callback: searchCb.focusNextResult },
+    { key: Keys.K, callback: searchCb.focusPrevResult },
+    { key: Keys.ARROW_UP, callback: searchCb.focusPrevResult },
+    { key: Keys.H, callback: searchCb.focusSearchBar },
+    { key: Keys.FORWARD_SLASH, callback: searchCb.focusSearchBar },
+    { key: Keys.V, callback: searchCb.openResultInNewTab },
+    { key: Keys.SINGLE_QUOTE, callback: searchCb.openResultInNewTab },
+    { key: Keys.T, callback: searchCb.scrollToTop },
+    { key: Keys.D, callback: searchCb.domainSearch },
+    { key: Keys.L, callback: searchCb.openResult },
+    { key: Keys.O, callback: searchCb.openResult },
+    { key: Keys.M, callback: searchCb.focusMainResult },
+    { key: Keys.S, callback: searchCb.goToMisspellLink },
+  ]);
```

Comment:
When `Key` is a type, this would be
```ts
const keybind = new Keybind([
    { key: 'j', callback: searchCb.focusNextResult },
    { key: 'ArrowDown', callback: searchCb.focusNextResult },
    { key: 'k', callback: searchCb.focusPrevResult },
    { key: 'ArrowUp', callback: searchCb.focusPrevResult },
    { key: 'h', callback: searchCb.focusSearchBar },
    { key: '/', callback: searchCb.focusSearchBar },
    { key: 'v', callback: searchCb.openResultInNewTab },
    { key: "'", callback: searchCb.openResultInNewTab },
    { key: 't', callback: searchCb.scrollToTop },
    { key: 'd', callback: searchCb.domainSearch },
    { key: 'l', callback: searchCb.openResult },
    { key: 'o', callback: searchCb.openResult },
    { key: 'm', callback: searchCb.focusMainResult },
    { key: 's', callback: searchCb.goToMisspellLink },
  ])
  ```